cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -55,29 +55,29 @@ The tips presented in this article are part of a Java project that was created t
 
 You can find the Java project [here](https://github.com/righettod/poc-jwt), it uses the official [JWT library](https://jwt.io/#libraries).
 
-In the rest of the article, the term **token** refer to the **JSON Web Tokens** (JWT).
+In the rest of the article, the term **token** refers to the **JSON Web Tokens** (JWT).
 
-# Consideration about using JWT
+# Consideration about Using JWT
```

Comment:
"About" should probably be capitalised for consistency

Reply:
"about" is a preposition. In the rules I learned for title casing, prepositions should be lowercase. See https://capitalizemytitle.com/ for list of rules for title casing.

Do you still think it should be capitalized? I have no strong preference either way.

Reply:
Just to add my two cents. Leaving it lower seems to be 'technically' correct based on the tool I normally use. https://capitalizemytitle.com/#Chicago

It does look out of place, I think due to the length of the word.

As long as the style is consistently adhered to I think that's the main thing. https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html seems to confirm that for Chicago style prepositions should be lower.

Reply:
@kingthorin I think I had exactly the same thought process as you - it looks _off_ based on the length of the word. If it's technically correct (according to some guide - and we don't have a consistent rule about which styles we use) then that's fine.

We've all got better things to do that argue about obscure grammar rules, so happy for this to stay as it is.